splink/predict.py

```diff
@@ -39,37 +39,17 @@ def predict_from_comparison_vectors_sqls(
 
     select_cols = settings_obj._columns_to_select_for_predict
     select_cols_expr = ",".join(select_cols)
-    mult = []
+    bf_terms = []
     for cc in settings_obj.comparisons:
-        mult.extend(cc._match_weight_columns_to_multiply)
+        bf_terms.extend(cc._match_weight_columns_to_multiply)
 
-    probability_two_random_records_match = (
-        settings_obj._probability_two_random_records_match
+    prior = settings_obj._probability_two_random_records_match
```

Comment:
@RobinL - do you care if this is changed from `probability_two_random_records_match` -> `prior`?

Imo it doesn't impact readability, but you may have a different view.

Reply:
Yeah looks fine to me